HW3.java

```diff
@@ -0,0 +1,119 @@
+public class HW3 {
+    //    Напишіть метод, який знаходить найдовше слово серед трьох,
+//    використовуючи if/else
+    public static void Word () {
+        // Довжина слова s1, s2, s3.
+        String s1 = "September";
+        int len1 = s1.length();
+        String s2 = "Nowember";
+        int len2 = s2.length();
+        String s3 = "May";
+        int len3 = s3.length();
+
+        // Виведення кількості символів
+        System.out.println( "Кількості символів \"September\": " + len1 + " символи.");
+        System.out.println( "Кількості символів \"Nowember\": " + len2 + " символи.");
+        System.out.println( "Кількості символів \"May\": " + len3 + " символи.");
+
+        // Порівняння слів s1, s2, s3.
+        if (len1 > len2) {
+            System.out.println( "\nСлово \"September\" Довше слова \"Nowember\".");
+        }
+        else if (len1 < len2) {
+            System.out.println( "\nСлово \"September\" Довше слова \"Nowember\".");
+        }
+        else if (len2 < len3) {
+            System.out.println("\nСлово \"Nowember\" Довше слова \"May\".");
+        }
+        else if (len2 > len3) {
+            System.out.println("\nСлово \"Nowember\" Довше слова \"May\".");
+        }
+        else if (len1 > len3) {
+            System.out.println( "\nСлово \"May\" Коротше слова \"September\".");
+        }
+        else if (len1 < len3) {
+            System.out.println( "\nСлово \"May\" Коротше слова \"September\".");
+        }
+        else {
+            System.out.println( "len1". length());
+            System.out.println( "len2". length());
+            System.out.println( "len3". length());
+        }
+    }
+    // У класі Month створити метод, у якого 1 параметр типу даних
+    //int. Цей параметр буде означати порядковий номер місяця.
+    //Використовуючи switch, вивести на екран кількість днів
+    //цього місяця (2023 рік). Перевірити роботу даного метода в
+    //main
+
+    public static String Day() {
```

Comment:
static int monthDayQuantity(int monthNumber) {
        switch (monthNumber) {
            case 1:
            case 3:
            case 5:
            case 7:
            case 8:
            case 10:
            case 12:
                System.out.println("31");
                break;
            case 2:
                System.out.println("28");
                break;
            case 4:
            case 6:
            case 9:
            case 11:
                System.out.println("30");
                break;
            default:
                System.out.println("Sorry, this value is out of the range, please enter correct month number");
        }
        return monthNumber;
    }